README.md

```diff
@@ -12,23 +12,22 @@ Tested with latest stable version.
 
 ```
 $ php bin/markbench benchmark --profile=github-sample
-Runtime: PHP5.5.9
-Host:    Linux testing-worker-linux-9-1-25355-linux-14-19501860 2.6.32-042stab079.5 #1 SMP Fri Aug 2 17:16:15 MSK 2013 x86_64
+Runtime: PHP5.5.21
+Host:    Linux testing-worker-linux-da6418d7-1-8430-linux-17-51028173 2.6.32-042stab090.5 #1 SMP Sat Jun 21 00:15:09 MSK 2014 x86_64
 Profile: Sample content from Github (http://github.github.com/github-flavored-markdown/sample_content.html) / 1000 times
 Class:   Markbench\Profile\GithubSampleProfile
-
-+----------------------+---------+---------+---------------+----------+--------------+
-| package              | version | dialect | duration (MS) | MEM (B)  | PEAK MEM (B) |
-+----------------------+---------+---------+---------------+----------+--------------+
-| erusev/parsedown     | 0.9.4   |         | 4183          | 9437184  | 9437184      |
-| cebe/markdown        | 0.9.2   |         | 5081          | 9437184  | 9699328      |
-| cebe/markdown        | 0.9.2   | gfm     | 6293          | 9437184  | 9699328      |
-| michelf/php-markdown | 1.4.0   |         | 15568         | 9699328  | 9699328      |
-| michelf/php-markdown | 1.4.0   | extra   | 23401         | 9699328  | 9961472      |
-| kzykhys/ciconia      | v1.0.3  |         | 32115         | 10747904 | 11010048     |
-| kzykhys/ciconia      | v1.0.3  | gfm     | 39654         | 10747904 | 11010048     |
-+----------------------+---------+---------+---------------+----------+--------------+
-```
++----------------------+---------+------------+---------------+----------+--------------+
+| package              | version | dialect    | duration (MS) | MEM (B)  | PEAK MEM (B) |
++----------------------+---------+------------+---------------+----------+--------------+
+| erusev/parsedown     | 1.5.1   |            | 6010          | 11010048 | 11010048     |
+| cebe/markdown        | 1.0.1   |            | 7695          | 11010048 | 11010048     |
+| cebe/markdown        | 1.0.1   | gfm        | 12925         | 11272192 | 11534336     |
+| michelf/php-markdown | 1.4.1   |            | 13952         | 11272192 | 11272192     |
+| michelf/php-markdown | 1.4.1   | extra      | 19476         | 11010048 | 11272192     |
+| kzykhys/ciconia      | v1.0.2  |            | 33115         | 11796480 | 11796480     |
+| kzykhys/ciconia      | v1.0.2  | gfm        | 40104         | 12320768 | 12582912     |
```

Comment:
Good question, I'm not really sure why Composer installed that version
instead.

On Tue Feb 17 2015 at 6:15:09 PM Carsten Brandt <notifications@github.com>
wrote:

> In README.md
> <https://github.com/kzykhys/Markbench/pull/11#discussion_r24864772>:
>
> > -| michelf/php-markdown | 1.4.0   |         | 15568         | 9699328  | 9699328      |
> > -| michelf/php-markdown | 1.4.0   | extra   | 23401         | 9699328  | 9961472      |
> > -| kzykhys/ciconia      | v1.0.3  |         | 32115         | 10747904 | 11010048     |
> > -| kzykhys/ciconia      | v1.0.3  | gfm     | 39654         | 10747904 | 11010048     |
> > -+----------------------+---------+---------+---------------+----------+--------------+
> > -```
> > ++----------------------+---------+------------+---------------+----------+--------------+
> > +| package              | version | dialect    | duration (MS) | MEM (B)  | PEAK MEM (B) |
> > ++----------------------+---------+------------+---------------+----------+--------------+
> > +| erusev/parsedown     | 1.5.1   |            | 6010          | 11010048 | 11010048     |
> > +| cebe/markdown        | 1.0.1   |            | 7695          | 11010048 | 11010048     |
> > +| cebe/markdown        | 1.0.1   | gfm        | 12925         | 11272192 | 11534336     |
> > +| michelf/php-markdown | 1.4.1   |            | 13952         | 11272192 | 11272192     |
> > +| michelf/php-markdown | 1.4.1   | extra      | 19476         | 11010048 | 11272192     |
> > +| kzykhys/ciconia      | v1.0.2  |            | 33115         | 11796480 | 11796480     |
> > +| kzykhys/ciconia      | v1.0.2  | gfm        | 40104         | 12320768 | 12582912     |
>
> how comes it uses 1.0.2 here while 1.0.3 was in the old table?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/kzykhys/Markbench/pull/11/files#r24864772>.
>